pandas/tests/frame/test_timeseries.py

```diff
@@ -336,3 +336,9 @@ def test_first_last_valid(self):
 
         index = frame.last_valid_index()
         self.assertEqual(index, frame.index[-6])
+
+    def test_range_in_days(self):
+        ts = pd.Series(pd.timedelta_range(pd.Timedelta.min,
+                                          pd.Timedelta.max,
+                                          freq='D'))
+        self.assertTrue(len(ts) != 0)
```

Comment:
More robust tests explicitly checking the values of `min`, `max` and `resolution` would be nice to have